transforms/images/polus-ftl-label-plugin/README.md

```diff
@@ -50,10 +58,49 @@ This plugin takes one input argument and one output argument:
 | `--connectivity` | City block connectivity                               | Input  | number     |
 | `--outDir`       | Output collection                                     | Output | collection |
 
+## Example Code
+```Linux
+cd examples
+mkdir output
+basedir=$(basename ${PWD})
+docker run -v ${PWD}:/$basedir labshare/polus-ftl-label-plugin:0.3.10 \
+--inpDir /$basedir/"images/" \
+--outDir /$basedir/"output/" \
+--connectivity 1
+```
```

Comment:
I see that data was included in this commit, but they should be removed because data shouldn't go into polus-data. Instead, this script should download a data set and run the plugin.

In addition, I think a one sentence explanation of what this script does would be helpful.

Reply:
Sorry, I was not aware of the no images policy. I'll see if I can re-work the script and add a few more sentences.